### PR TITLE
add an install target to the MakeFile

### PR DESCRIPTION
**Pull request by marius851000**
I wasn't able to test it system wide, as I can't install app system-wide on nixos (or at least it is unsupported). I however tested the installation on another folder, and it seem to work.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -518,6 +518,51 @@ $(LINUX_DEB_DIR)/data/usr/share/icons/hicolor/%/apps/azimuth.png: \
...
+	mkdir -p $(INSTALLICONDIR)/128x128/apps/ $(INSTALLICONDIR)/64x64/apps/ $(INSTALLICONDIR)/48x48/apps/ $(INSTALLICONDIR)/32x32/apps/
+	install data/icons/icon_128x128.png $(INSTALLICONDIR)/128x128/apps/azimuth.png
+	install data/icons/icon_64x64.png $(INSTALLICONDIR)/64x64/apps/azimuth.png
+	install data/icons/icon_48x48.png $(INSTALLICONDIR)/48x48/apps/azimuth.png
+	install data/icons/icon_32x32.png $(INSTALLICONDIR)/32x32/apps/azimuth.png
+	mkdir -p $(INSTALLSHAREDIR)/applications
+	cp data/azimuth.desktop $(INSTALLSHAREDIR)/applications/azimuth.desktop
+
+
+uninstall:
```
Comment (mdsteele) [inline Makefile]: The `uninstall` target should also be marked `.PHONY`, right?

Comment (marius851000) [reply Makefile]: That's true. I've also make the tool not install by default.

**Comment by marius851000** [comment]
Can you make a news release, please.

**Comment by mdsteele** [comment]
Sure, tagged as [v1.0.3](https://github.com/mdsteele/azimuth/releases/tag/v1.0.3).

**Pull request merged**